app/v2/connectors/CreateUkPropertyPeriodSummaryConnector.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.connectors
+
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import play.api.http.Status
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+import v2.connectors.httpparsers.StandardIfsHttpParser._
+import v2.models.request.createUkPropertyPeriodSummary.CreateUkPropertyPeriodSummaryRequest
+import v2.models.response.createUkPropertyPeriodSummary.CreateUkPropertyPeriodSummaryResponse
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class CreateUkPropertyPeriodSummaryConnector @Inject()(val http: HttpClient,
+                                                       val appConfig: AppConfig) extends BaseIfsConnector {
+
+  def createUkProperty(request: CreateUkPropertyPeriodSummaryRequest)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext,
+    correlationId: String): Future[IfsOutcome[CreateUkPropertyPeriodSummaryResponse]] = {
+
+    implicit val successCode: SuccessCode = SuccessCode(Status.OK)
+
+    post(
+      body = request.body,
+      uri = IfsUri[CreateUkPropertyPeriodSummaryResponse](s"income-tax/business/property/annual?taxableEntityId=${request.nino.nino}&incomeSourceId=${request.businessId}&taxYear=${request.taxYear}")
```

Comment:
incorrect uri, this should be periodic, not annual

`/income-tax/business/property/periodic?taxableEntityId={taxableEntityId}&taxYearExplicit={taxYearExplicit}&incomeSourceId={incomeSourceId}`